utils/create-jobs.py

```diff
@@ -43,14 +45,14 @@ def main():
 
     if args.job_id is not None:
         args.urlbase = urlparse.urljoin(args.urlbase, args.job_id + '/')
-        args.job_name += ' - {}'.format(args.job_id)
+        args.job_name += '-{}'.format(args.job_id)
```

Comment:
This and the following changes look like coding style correction which is good. It would be better to put them in another patch though